src/Xrefcheck/Verify.hs

```diff
@@ -255,6 +261,24 @@ instance Given ColorMode => Buildable VerifyError where
         #{redirectedUrl}
       |]
 
+    RedirectCycle _ ->
+      [int||
+      Cycle found in the following redirect chain:
+      TODO print chain
```

Comment:
I'm thinking about how to show a redirect chain of links. How about this?

```
Cycle found in the following redirect chain:
  -| http://github.com/some
  -> https://github.com/some
  -> https://github.com/other
```

Reply:
Wow, cool. I like it.